kuma/static/js/wiki.js

```diff
@@ -825,7 +825,7 @@
                     });
                 });
                 players[i].addEventListener('onError', function(event) {
-                    mdn.analytics.trackError('YouTube Error: ' + event.data, win.location.href);
+                    mdn.analytics.trackError('YouTube Error', win.location.href, event.data);
```

Comment:
Same. If there's another interesting or helpful property of the `event` that we want to store as the event `action` we can do that.